src/components/AddItem.js

```diff
@@ -38,10 +27,10 @@ const AddItem = () => {
       .then((data) => {
         if (data.data()) {
           if (existingName(data.data(), newItem.name)) {
-            displayMessage(
```

Comment:
What do you think about instead of removing **displayMessage**, move the **Bus.emit** logic inside the displayMessage function. This can have 2 benefits, 

1. If at some point you want to change the implementation, you can keep the interface and only change the **displayMessage** function ( touching only one file / function ). 
2. About semantics, **displayMessage** vs **Bus.emit('flash')**, the first seems clearer

Reply:
I really like the idea of using an EventEmitter to display your flash messages. But it would be even better - as David suggested - that you move the Bus.emit inside the displayMessage function.

Reply:
Yeah agree with you that will make it more readable